bot/connector-whatsapp-cloud/pom.xml

```diff
@@ -79,6 +79,14 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.litote.kmongo</groupId>
+            <artifactId>kmongo</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.litote.kmongo</groupId>
+            <artifactId>kmongo-async</artifactId>
+        </dependency>
```

Comment:
remove kmongo-async - it should work with kmongo only

Reply:
Just wondering, what's the reason ?